Makefile

```diff
@@ -135,15 +135,18 @@ $(java_kompiled): $(java_files)
 # Test
 # ----
 
+KMCD_GENDEPTH   := 20
+KMCD_RANDOMSEED := 0
```

Comment:
Fixing the seed across runs means that the sequence of random numbers generated will be the same every time the tests are run, which can be useful when one needs to reproduce results. Another option here is to use the system clock time value (I suppose something like `date +%s`) as the seed, so that a fresh set of executions are generated at every run.

Reply:
In a later PR I select out a particular seed for each run as a "witness". This is for testing, so we want deterministic runs.